subprojects/packagefiles/google-highway/meson.build

```diff
@@ -0,0 +1,155 @@
+project(
+  'highway',
+  'cpp',
+  version: '1.2.0',
+  license: 'Apache-2.0',
+)
+
+highway_sources = files(
+    'hwy/abort.cc',
+    'hwy/aligned_allocator.cc',
+    'hwy/nanobenchmark.cc',
+    'hwy/per_target.cc',
+    'hwy/print.cc',
+    'hwy/targets.cc',
+    'hwy/timer.cc',
+)
+
+if host_machine.system() == 'windows'
+  add_project_arguments('-DNOMINMAX', language: 'cpp')
+endif
+is_shared = (get_option('default_library') != 'static')
+args = is_shared ? ['-DHWY_SHARED_DEFINE', '-Dhwy_EXPORTS'] : ['-DHWY_STATIC_DEFINE']
+
+lib_highway = library(
+    'highway',
+    highway_sources,
+    cpp_args: args,
+    gnu_symbol_visibility: 'hidden',
+)
+
+inc = include_directories('.')
+
+google_highway_dep = declare_dependency(
+    link_with: lib_highway,
+    include_directories: inc,
+)
+
+if not get_option('tests').disabled()
+  highway_test_sources = [
+      'hwy/tests/hwy_gtest.h',
+      'hwy/tests/test_util-inl.h',
+      'hwy/tests/test_util.cc',
+      'hwy/tests/test_util.h',
```

Comment:
Headers are not needed here, they'll just be dropped anyway.

Reply:
Sounds good - was just trying to keep this in sync with the upstream CMake configuration, but assuming that's not a big deal